netdata-installer.sh

```diff
@@ -4,29 +4,25 @@
 
 export PATH="${PATH}:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin"
 uniquepath() {
-    local path=""
-    while read
-    do
-        if [[ ! "${path}" =~ (^|:)"${REPLY}"(:|$) ]]
-        then
-            [ ! -z "${path}" ] && path="${path}:"
-            path="${path}${REPLY}"
-        fi
-    done < <( echo "${PATH}" | tr ":" "\n" )
-
-    [ ! -z "${path}" ] && [[ "${PATH}" =~ /bin ]] && [[ "${PATH}" =~ /sbin ]] && export PATH="${path}"
+	local path=""
+	while read; do
+		if [[ ! ${path} =~ (^|:)"${REPLY}"(:|$) ]]; then
+			[ ! -z "${path}" ] && path="${path}:"
+			path="${path}${REPLY}"
+		fi
+	done < <(echo "${PATH}" | tr ":" "\n")
+
+	[ ! -z "${path}" ] && [[ ${PATH} =~ /bin ]] && [[ ${PATH} =~ /sbin ]] && export PATH="${path}"
```

Comment:
quotes from ${PATH}` removed.
is that ok?

Reply:
yes it is